dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantVectorStore.cs

```diff
@@ -65,14 +65,21 @@ internal QdrantVectorStore(MockableQdrantClient qdrantClient, QdrantVectorStoreO
         };
     }
 
+    /// <inheritdoc/>
+    protected override void Dispose(bool disposing)
+    {
+        this._qdrantClient.Dispose();
+        base.Dispose(disposing);
+    }
+
 #pragma warning disable IDE0090 // Use 'new(...)'
     /// <inheritdoc />
 #if NET8_0_OR_GREATER
     public override QdrantCollection<TKey, TRecord> GetCollection<TKey, TRecord>(string name, VectorStoreRecordDefinition? vectorStoreRecordDefinition = null)
 #else
     public override VectorStoreCollection<TKey, TRecord> GetCollection<TKey, TRecord>(string name, VectorStoreRecordDefinition? vectorStoreRecordDefinition = null)
 #endif
-        => new QdrantCollection<TKey, TRecord>(this._qdrantClient, name, new QdrantCollectionOptions()
+        => new QdrantCollection<TKey, TRecord>(this._qdrantClient.IncreaseReferenceCount(), name, new()
```

Comment:
Slightly weird to have IncreaseReferenceCount() return but OK :) I'd very slightly prefer to just have a separate call for that, but feel free to ignore.